src/core/bindEvents.js

```diff
@@ -33,8 +33,12 @@ function bindEvents(){
                 `.${pagination.pageClass}`,
                 this.paginationAction.bind(this)
             )
-            //wrapper.addEventListener(pagination.action, this.paginationAction.bind(this));
         });
+        if(pagination.type === 'INFINITE_SCROLL') {
+            pagination.infinteScrollTriggerEl.addEventListener("scroll",debounce(()=>{
```

Comment:
check if `infinteScrollTriggerEl` is defined before invoking `addEventListener` on it

Reply:
by default it is window.